libbpf-cargo/src/btf/btf.rs

```diff
@@ -577,18 +581,34 @@ impl<'a> Btf<'a> {
         }))
     }
 
-    fn load_struct(&self, t: &btf_type, extra: &'a [u8]) -> Result<BtfType<'a>> {
+    fn load_struct(&mut self, t: &btf_type, extra: &'a [u8]) -> Result<BtfType<'a>> {
+        let name = match self.get_btf_str(t.name_off as usize)? {
+            "" => {
+                let n = format!("anon_struct_{}", self.anon_struct_count);
```

Comment:
Nit: let's do `__anon_struct_{}` instead

We already have `__pad_{}` so it'd match